5.5/Dockerfile

```diff
@@ -36,6 +33,9 @@ RUN rpmkeys --import file:///etc/pki/rpm-gpg/RPM-GPG-KEY-CentOS-7 && \
     test "$(id mysql)" = "uid=27(mysql) gid=27(mysql) groups=27(mysql)" && \
     chmod o+w -R /var/lib/mysql
 
+COPY run-*.sh /usr/local/bin/
+COPY contrib /var/lib/mysql/
```

Comment:
same comment as the other PR (but that chmod o+w should be a chmod a+rwx i think.  or at least a+rw)